examples/ffi/rust_calling_c/c/BUILD

```diff
@@ -20,9 +20,9 @@ cc_test(
 
 ## Do the same as above, but with a dynamic c library.
 
-cc_library(
+cc_import(
```

Comment:
Can you also update the load statement to include `cc_import`?

Reply:
Done.